recipe/meta.yaml

```diff
@@ -10,14 +10,14 @@ source:
   url: https://ftp.postgresql.org/pub/source/v{{ version }}/postgresql-{{ version }}.tar.bz2
   sha256: {{ sha256 }}
   patches:
-    - 0001-fix-win-zlib-name.patch
-    - 0002-disable-new-dtags.patch
-    - 0003-update-location-of-krb5-files.patch
-    - 0004-fix-windows-openssl-links-for-openssl-1.1.1.patch  # [win]
-    - 0005-change-setenv-on-windows.patch                     # [win]
+    - patches/0002-disable-new-dtags.patch
+    - patches/fix_gssapi_setenv_win.patch       # [win]
+    - patches/fix_auth_setenv_win.patch         # [win]
+    - patches/fix_x509_name_win.patch           # [win]
+    - patches/fix_mac10_9_clock_realtime.patch  # [osx]
 
 build:
-  number: 0
+  number: 4
```

Comment:
We can reset to 0 in this case. We usually only try to keep the same build number when forking recipes. But once it's forked and we already released versions, we already diverged from CF so we can use our own build numbers.